.github/workflows/merge-gatekeeper.yaml

```diff
@@ -0,0 +1,19 @@
+name: "*Merge Gatekeeper"
+
+on:
+  pull_request:
+
+jobs:
+  merge-gatekeeper:
```

Comment:
Can we gate this to user / branch for some time first?

For instance gate to our prod eng group or a set of usernames

Reply:
no easy way to do this, unless we block on a specific set of branch prefixes. I think it's worth getting the signal everywhere for now

Reply:
Cant we do something silly to start? at least for a couple days, opt in some blockchain people and our team

```
          if [[ "${{ github.actor }}" == "your-username" ]]; then
```

Reply:
ok lfg, always down for silly